orca-plugins-test/src/test/kotlin/com/netflix/spinnaker/orca/plugins/test/OrcaPluginsFixture.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.plugins.test
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.netflix.spinnaker.kork.plugins.SpinnakerPluginManager
+import com.netflix.spinnaker.kork.plugins.internal.PluginJar
+import com.netflix.spinnaker.kork.plugins.tck.PluginsTckFixture
+import com.netflix.spinnaker.orca.Main
+import com.netflix.spinnaker.orca.StageResolver
+import com.netflix.spinnaker.orca.clouddriver.service.JobService
+import com.netflix.spinnaker.orca.notifications.NotificationClusterLock
+import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository
+import com.netflix.spinnaker.orca.plugins.OrcaPlugin
+import com.netflix.spinnaker.orca.plugins.PreconfiguredJobConfigurationProviderExtension
+import com.netflix.spinnaker.orca.plugins.SimpleStageExtension
+import com.netflix.spinnaker.orca.plugins.StageDefinitionBuilderExtension
+import com.netflix.spinnaker.orca.plugins.TaskExtension1
+import com.netflix.spinnaker.orca.plugins.TaskExtension2
+import com.netflix.spinnaker.q.Queue
+import com.netflix.spinnaker.q.memory.InMemoryQueue
+import com.netflix.spinnaker.q.metrics.EventPublisher
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Primary
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.TestPropertySource
+import java.io.File
+import java.time.Clock
+import java.time.Duration
+
+class OrcaPluginsFixture : PluginsTckFixture, OrcaTestService() {
+  val objectMapper: ObjectMapper = ObjectMapper(YAMLFactory())
+    .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
```

Comment:
Should this use `OrcaObjectMapper`?

Reply:
`OrcaObjectMapper` uses the standard `JsonFactory`, this needs to use the `YAMLFactory`.  We don't really have a use for an ObjectMapper that uses the `YAMLFactory` in Orca, except for in this test to verify configs are loaded correctly.  So I think it's fair to just leave this here.